MK404.cpp

```diff
@@ -140,6 +140,16 @@ void KeyCB(unsigned char key, int /*x*/, int y)
 	}
 }
 
+void SpecialKeyCB(int key, int x, int y)
+{
+	if (key==0x65) { // Up
```

Comment:
There are `GLUT_KEY_UP` and `GLUT_KEY_DOWN` #defines that are probably better used here rather than magic numbers